### ISSUE TITLE
Update docs to include GitHub Actions example

### ISSUE THREAD

**Issue by charleskawczynski** [bors-ng/bors-ng#926]
I think it would be useful to add a GitHub Actions example

**Comment by CleanCut** [comment]
Yes, what is the magic incantation?  I can't get my status to match.

**Comment by charleskawczynski** [comment]
I think the format depends on the type of job. As an example, here is a [bors toml](https://github.com/CliMA/CalibrateEmulateSample.jl/blob/master/bors.toml), and GHA yaml file which, IIUC, needs to match [here](https://github.com/CliMA/CalibrateEmulateSample.jl/blob/master/.github/workflows/Tests.yml#L10) and [here](https://github.com/CliMA/CalibrateEmulateSample.jl/blob/master/.github/workflows/Tests.yml#L64).

**Comment by CleanCut** [comment]
@charleskawczynski Thanks! I've tried matching the entry to both the job (`tests:` in GHA -> `tests` in bors) and the name (`name: Tests` in GHA -> `Tests` in bors), neither of which have worked. 😢 Perhaps it would work if I removed the `name: Tests` entry from GHA and tried `tests` again in bors. That would be most similar to what you have.

**Comment by CleanCut** [comment]
Aha! Someone pointed out to me that [GitHub apps (like bors) can't merge PRs that change workflows](https://forum.bors.tech/t/resource-not-accessible-by-integration/408/7).  If true, this would have saved me many hours of work if bors would have simply crashed with some error to that effect rather than crashing silently.  I'm going to try manually merging my setup and then see if bors will work on _other_ PRs afterwards.

**Comment by charleskawczynski** [comment]
Sounds like another good thing to add to the docs. Perhaps this restriction generally makes GHA CI not very compatible with Bors.

**Comment by CleanCut** [comment]
I don't see why it would make it incompatible, but it would definitely help to:
- Document that bors can't merge PRs that modify workflow files
  - Expose the error to the user if it crashes due to this situation
- Document a GHA example
  - Point out that the CI workflow needs to trigger for pushes to `staging` and `trying` -- I figured that out only by much reading through the docs
  - Point out the details of how to match bors up to workflow job names
